scenes/test_object.txt

```diff
@@ -0,0 +1,45 @@
+*label entry
+*params module log_buffer
+*temp test_count 0
+*temp tests_passed 0
+
+*gosub test_start "get unmodified data"
+*gosub_scene cslib_object get "testobj" 1
+*gosub test_assert (testobj_testfield1 = "Calvin") "=> Expected - Calvin[n/]=> Received - ${testobj_testfield1}[n/]"
```

Comment:
With this one you can use `testobj_testfield1  = testobj_1_testfield1` rather than a hardcoded "Calvin". That way you don't have to update both the test and startup.txt with a new value, you could just modify startup.txt.

Reply:
I think it also makes sense to assert both fields to catch any off-by-1 errors?

Reply:
Makes sense